autogpt_platform/backend/backend/server/v2/store/db.py

```diff
@@ -696,45 +696,35 @@ async def get_my_agents(
     logger.debug(f"Getting my agents for user {user_id}, page={page}")
 
     try:
-        agents_with_max_version = await prisma.models.AgentGraph.prisma().find_many(
-            where=prisma.types.AgentGraphWhereInput(
-                userId=user_id, StoreListing={"none": {"isDeleted": False}}
-            ),
-            order=[{"version": "desc"}],
-            distinct=["id"],
-            skip=(page - 1) * page_size,
-            take=page_size,
+        search_filter = prisma.types.LibraryAgentWhereInput(
```

Comment:
Please use normal dictionary syntax for this, like
```suggestion
        search_filter: prisma.types.LibraryAgentWhereInput = {
```

Reply:
I'll do this with the other PRs

Reply:
Addressed here:
 https://github.com/Significant-Gravitas/AutoGPT/pull/9617